.travis.yml

```diff
@@ -1,8 +1,6 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
```

Comment:
These are removed because Laravel 5 doesn't support PHP 5.3 and the tests use `::class` which isn't supported in PHP 5.4